charts/kubewarden-controller/values.yaml

```diff
@@ -205,7 +205,7 @@ auditScanner:
     # The registry is defined in the common.cattle.systemDefaultRegistry value
     # kubectl image to be used in the pre-delete helm hook
     repository: "kubewarden/audit-scanner"
-    tag: v1.15.0
+    tag: v1.15.1
```

Comment:
This version does not exist in the repo yet.

Reply:
mmh, true. This is a bug in the automation:
https://github.com/kubewarden/helm-charts/actions/runs/10181224959/job/28160875951#step:9:72

Reply:
Opened https://github.com/kubewarden/helm-charts/issues/500.

Reply:
Fixed manually with https://github.com/kubewarden/helm-charts/pull/499/commits/ace9c9c50b3ca52a651af4c003e3dc9dfa99b886.

Reply:
now, bumped to audit-scanner v1.15.1 because it is out.